src/bin/wallet.rs

```diff
@@ -155,7 +155,7 @@ fn parse_args() -> Result<WalletConfig, Box<error::Error>> {
         path.to_str().unwrap()
     };
     let id = read_keypair(id_path).or_else(|err| {
-        display_actions();
+        println!("{}", matches.usage());
```

Comment:
I'm not sure that this print statement actually provides anything of value here (about avoiding a keypair error). Maybe just remove it entirely?